sw/device/rom_exts/manifest.rs.tpl

```diff
@@ -0,0 +1,18 @@
+/// Copyright lowRISC contributors.
+/// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+/// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Can you include a module comment (i.e. //! comments).

Reply:
He wants a module comment, which says what the module contains. Kind-of like the `/** @file … */` comments we use on C/C++ files.

Reply:
I am not opposed to this, but it is not clear to me what information should be added, happy to fire a follow-up. For now I think it is fine to be merged.